cdap-docs/reference-manual/source/http-restful-api/logging.rst

```diff
@@ -163,6 +163,8 @@ A filter string can be supplied as an optional parameter. It will filter the ret
 to those that match the supplied string.
 
 
+.. _http-restful-api-logging-changing-program-log-levels:
+
 Changing Program Log Levels
 ===========================
 Log levels can be set for a particular run of a program. Once changed, they can be reset back to what
```

Comment:
This is only supported for program type flow, service or worker. The corresponding programType at Line 210 should be updated accordingly